2013/data_file_specification.json

```diff
@@ -260,6 +390,9 @@
                "end": "54",
                "length": "5",
                "dataType": "N",
+               "format": "currency,thousands",
```

Comment:
@awolfe76 my main justification for using an object over an array is that then the spec is self-documenting. If we used an array then we would want to provide additional documentation about what each position in the array meant. Whereas if we use an object then we can use the key to describe the value; keeping all of the documentation in the spec.

I'm also still struggling with what to call that extra formatting, but by making the format value an object we could make the keys descriptive of our intent. For example with the currency we could have:

```json
format: {
    type: 'currency',
    roundTo: 'thousands' 
}
```

while maybe the datetime item above could be:

```json
format: {
    type: 'datetime',
    toString: 'yyyy/MM/dd HH:mm' 
}
```